react/RedirectList.tsx

```diff
@@ -219,6 +218,20 @@ const RedirectList: React.FC<Props> = ({
           const redirects = data?.redirect?.listRedirects.routes || []
           const hasRedirects = redirects.length > 0
 
+          const handleInputSearchChange = (
+            e: React.ChangeEvent<HTMLInputElement>
+          ) => {
+            const filteredRedirects = redirects.filter(item =>
+              item.from.includes(e.target.value)
```

Comment:
Hi! One suggestion. Maybe will be interested that the search don't be case sensitive.

<img width="641" alt="Screenshot 2023-06-02 at 12 10 26" src="https://github.com/vtex-apps/admin-pages/assets/21042186/d1012f39-419e-4744-8b9c-9d125f08d664">
<img width="739" alt="Screenshot 2023-06-02 at 12 12 43" src="https://github.com/vtex-apps/admin-pages/assets/21042186/1197670d-356c-472c-90d3-3b7722f45371">